crates/net/network/src/discovery.rs

```diff
@@ -43,6 +48,38 @@ pub struct Discovery {
     queued_events: VecDeque<DiscoveryEvent>,
 }
 
+/// An entry in the discovery map. It contains the latest known info
+/// about a node, and a
+pub struct DiscoveryEntry {
+    remote_addr: SocketAddr,
+    fork_id: Option<ForkId>,
+    node_record: NodeRecord,
+    bond_expiry: std::time::Instant,
+}
```

Comment:
this seems redundant to me because we already have all of this in the peersmanager's peerset.

ptal at my comment here https://github.com/paradigmxyz/reth/issues/1415#issuecomment-1582634873